pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>javacc21</groupId>
+	<artifactId>javacc</artifactId>

```

Comment:
Do you like?

Reply:
Looks okay to me. But I honestly don't have much criteria for judging these things. My main concern now is just to be able to automatically publish the full-jar to Maven Central because, for some people this is important. Maybe description can be: "The most advanced version of the well known JavaCC Parser Generator, originally from Sun Microsystems". I honestly don't know what sort of text is supposed to be in the description.